R/secure_rmd.R

```diff
@@ -104,20 +107,42 @@ secure_rmd <- function(
   yaml_header <- yamlFromRmd(rmd_file_path)
 
   yaml_polished <- yaml_header$polished
```

Comment:
if we change this to

yaml_polished <- as.list(yaml_header$polished)

and the next one to

yaml_polished_global_config <- as.list(yaml_polished$global_sessions_config)

I think this fixes an error when "polished" is not included in the YAML header

Reply:
Shouldn't this just come as `NULL`?  Does `modifyList` not work with `NULL`?

Reply:
yea unfortunately ModifyList does not work with NULL